_includes/newsletter.html

```diff
@@ -0,0 +1,18 @@
+<script type="text/javascript" src="//downloads.mailchimp.com/js/signup-forms/popup/embed.js" data-dojo-config="usePlainJson: true, isDebug: false">
```

Comment:
I suggest removing `type="text/javascript"` (not needed in HTML5) and making the requested URL specifically over HTTPS (src="https://downloads....), its potentially safer.

Reply:
Thanks for the feedback @coliff . That specific line came straight from Mailchimp's 'pop-up code'. I've customized the rest of the code to make it work on-click and not automatically. 
Independently, true, bringing HTTPS in and 'type' won't hurt. More comments from me below.